analytics/cdk/emr-eks/lib/emr-eks-blueprint-stack.ts

```diff
@@ -0,0 +1,93 @@
+import { Construct } from 'constructs';
+import * as blueprints from '@aws-quickstart/eks-blueprints'
```

Comment:
Generally across this PR, please avoid importing `*` in favor of selective imports as done below, and additionally remove unused imports.